src/lib/datetimeUtils.ts

```diff
@@ -33,9 +33,22 @@ export const firstOfMonthStr = (dateStr: string) => {
 	return m[1] + '01';
 };
 
+const PhilippinesTimezoneOffset = -480;
+
+export const toPanglaoTimezone = (dt: Date) => {
+    let local = dt.getTimezoneOffset();
+    if (local == PhilippinesTimezoneOffset) {
+        return dt;
+    } else {
+        return new Date(dt.getTime() - (PhilippinesTimezoneOffset - local) * 60000);
+    }
+}
```

Comment:
we should really avoid building our own long form utilities.

this can work with:
```
export const toPanglaoTimezone => dayjs.tz(dt, "Asia/Manila")
```

Reply:
Ok I updated it to use dayjs.tz in the latest commit.  Note that two new packages, utc and timezone, have to be installed for it to work properly (package.json is updated to reflect this).

The new version with dayjs still returns a Date object because the Date and dayjs obect apis are not compatible, so returning a dayjs obect would require updating large portions of the codebase.  We can create a separate 'switch to dayjs for datetime objects' task for that work.

I also realized that, while you can change the time of the Date/dayjs object, I don't think you can actually change the timezone that the object thinks it's in, i.e. it's not possible to determine if an object has already been converted to Panglao time based on its timezone field.  Therefore I think it's only reasonable to do the timezone conversion on date objects that are instantiated within the PanglaoTime function so you can be certain that the object hasn't been converted to Panglao time already.  I updated the code to work like this.